docs/user/dashboard/edit-dashboards.asciidoc

```diff
@@ -0,0 +1,113 @@
+[[edit-dashboards]]
+== Edit dashboards
+
+Now that you have added panels to your dashboard, you can add interactive filter panels, Markdown panels, drill down into the  and customize the layout.  
```

Comment:
I'm not sure what the copy is for 'input controls', but I haven't heard of them called 'interactive filter panels' before. Below they are referred to as 'controls' panels.

Also - "...Markdown panels, drill down into the  and customize the layout." seems to be missing a word. I would consider re-wording it as something like "use Markdown panels to further customize the layout, or use drilldowns to dynamically link to other dashboards."